src/app/booking/OOSPlannerSection.js

```diff
@@ -331,9 +456,13 @@ function FamilyBookings() {
               </Button>
             </StyledTableCell>
           </TableRow>
+          {children && (
+            <FriendsBookings dataChildren={children} weekDates={weekDates} />
```

Comment:
<img width="926" alt="image" src="https://github.com/digital-brother/ooscca-fe/assets/43747573/c94a5ee7-31e4-4a48-b6c2-6018a8d4e468">

Comment not resolved